package.json

```diff
@@ -15,19 +15,32 @@
     "dependencies": {
         "@prisma/client": "^4.11.0",
         "altair-express-middleware": "^5.0.17",
+        "apollo-server-core": "^3.12.0",
         "apollo-server-express": "^3.12.0",
+        "bcrypt": "^5.1.0",
+        "bcryptjs": "^2.4.3",
```

Comment:
- Is it ok, that the package you're using [hasn't been updated for 6 years](https://www.npmjs.com/package/bcryptjs)?
- Why do you need two packages for the same thing if you're using only one?

Reply:
ditched the `.js` one.